### ISSUE TITLE
TypeError: the JSON object must be str, not 'bytes'

### ISSUE THREAD

**Issue by 5t4rdu5t** [uinput/deeplator#4]
I've done the install with : 

```bash
sudo python3 setup.py install
```
(no error) then try the _hello world_ example : 

```
$ cat  /tmp/file.py
from deeplator import Translator

t = Translator("EN", "DE")
sentence = "Hello world."
translation = t.translate_sentence(sentence)
print(translation)
```

Then run it : 

```
$ python3 !$
python3 /tmp/file.py
Traceback (most recent call last):
  File "/tmp/file.py", line 5, in <module>
    translation = t.translate_sentence(sentence)
  File "/usr/local/lib/python3.5/dist-packages/deeplator-0.0.4-py3.5.egg/deeplator/translator.py", line 113, in translate_sentence
  File "/usr/local/lib/python3.5/dist-packages/deeplator-0.0.4-py3.5.egg/deeplator/translator.py", line 84, in translate_sentences
  File "/usr/local/lib/python3.5/dist-packages/deeplator-0.0.4-py3.5.egg/deeplator/translator.py", line 131, in _send_jsonrpc
  File "/usr/local/lib/python3.5/dist-packages/deeplator-0.0.4-py3.5.egg/deeplator/jsonrpc.py", line 30, in send
  File "/usr/lib/python3.5/json/__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'bytes'
```

Any hint ?

**Comment by sputnick-dev** [comment]
It's a debian 9 for information upper

**Comment by winpassuser** [comment]
Open jsonrpc.py and change line 30 into:

`resp = json.loads(data_str.decode('utf-8'))`

**Comment by uinput** [comment]
Should be fixed in 538fc52461d1ed34df17df2b06841952737012f0. Could you verify if it works now?

**Comment by 5t4rdu5t** [comment]
Yes, it works, thanks
